### DISCUSSION TITLE
Student dev pack resolves email to wrong college

### DISCUSSION THREAD

**Discussion by k7plc** [community/community#17732] [GitHub Education]
I am a student of Clark College in Vancouver Washington. Our student email domain is @students.clark.edu. When attempting to apply for the Student Developer Pack, the automated system resolves this domain as Clarke College which is in Iowa, not Washington. 

Then, when you submit your valid student enrollment proof, the system denies you. When you file a support ticket you get _"It seems that you have applied from a location that is far from your campus. We will need you to reapply from the campus."_ Trying to explain the issue to support proves ineffective and no progress gets made towards any kind of resolution, either a manual approval or a fix of the backend domain-to-school name resolver. 

Is there an admin monitoring this forum who can raise the issue internally to fix the backend lookup? 
If helpful, the support incident number is 1590399.

**Comment by elisemoe** [comment]
Oh no! Thank you for sharing with us @epiccitizen! ✨  The database is updated and you should be able to reapply. Let us know if you run into any trouble.

**Reply by k7plc** [reply]
Thank you for correcting this! It appears my application has been approved. I appreciate your help!
